src/write_gcm.c

```diff
@@ -3019,25 +3020,16 @@ static int wg_json_CreateTimeSeries(
                 "type_instance: %s metric type must be string.",
                 head->key.plugin, head->key.plugin_instance, head->key.type,
                 head->key.type_instance);
-          continue;
-        }
-        const char *pref = custom_metric_prefix;
-        if (strncmp(entry->value.value_text, pref, strlen(pref)) != 0) {
-          ERROR("write_gcm: plugin: %s, plugin_type: %s, metric_type: %s, "
-                "type_instance: %s metric type %s is not a custom metric "
-                "(should start with '%s').",
-                head->key.plugin, head->key.plugin_instance, head->key.type,
-                head->key.type_instance, entry->value.value_text, pref);
-          continue;
+          continue next_payload;
```

Comment:
Yes. That's what the "make the metric_type check effective" refers to in the PR description.